### ISSUE TITLE
Staging Review finding: Confusing labeling for search input

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#99046]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** 10-10 Health Apps
**Product:** 10-10CG
**Feature:** Improve Facility Selection

## Findings details
**VA.gov Experience Standard - issue:** [User encounters no repetition or redundancy.](https://design.va.gov/about/experience-standards/#user-encounters-no-repetition-or-redundancy)
**VA.gov Experience Standard - category:** [Efficiency](https://design.va.gov/about/experience-standards/#category-efficiency)
**Launch-blocking:** No
**Design System review:** Yes
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

On [/locator/](https://staging.va.gov/family-and-caregiver-benefits/health-and-disability/comprehensive-assistance-for-family-caregivers/apply-form-10-10cg/veteran-information/va-medical-center/locator), the way the search input is labeled may lead to some confusion for screen reader users.

The `va-search-input` component has a built-in screen reader-only label. (This makes sense when the component is used for sitewide search in the header, for example, but maybe should be reexamined by the Design System Team for use cases like this.) That label reads: "City, state or postal code," which is coming from the `label` prop on the component.

Inside the `va-search-input` component there's also an `aria-label` on the input itself that reads "City, state or postal code." I believe this is a bug in the component, which I'll be passing along to the Design System Team.

There's also a separate `label` element that appears just before the compoinent. The `label` reads "City, state or postal code (* Required)" and includes the attribute `for="facility-search"`. As far as I can tell, there's no element on the page with ID `facility-search`, so this label isn't programmatically connected to anything.

If I'm a screen reader user and I have my screen reader announce the whole content of the page, I'll hear: "City, state or postal code (* Required). Search input, City, state or postal code. City, state or postal code."

If I'm a screen reader user and I tab directly to the input itself, I'll hear "Search input, City, state or postal code" --- so no repetition, but also no mention of it being a required field.

<details><summary>Link, screenshot or steps to recreate</summary>

![Facility locator page with a card containing a search input labeled "City, state, or postal code (* Required)."](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/51980-Brian-1.png?raw=true)

Coded as:
```
<va-card role="search" background="true" class="hydrated">
   <div class="">
      <label for="facility-search" class="vads-u-margin-top--0 vads-u-margin-bottom--1">City, state or postal code<span class="vads-u-color--secondary-dark"> (*Required)</span></label>
       <va-search-input label="City, state or postal code" value="" uswds="true" class="hydrated"></va-search-input>
   </div>
</va-card>
```

</details>

## Recommended action

There are multiple things happening here, including a bug in the component.

For the 10-10 team: 
- The `label` separate from the `va-search-input` component is important for sighted users to understand what's happening here, but it's redundant for screen reader users. Best option is to just make it a `p` element (since it's not programmatically labeling anything) with `aria-hidden="true"`. That will hide that line of text from screen readers and cut down on the redundancy.
- Add "(* Required)" to the `label` prop for the `va-search-input` component. This will ensure that screen reader users know that an input is required.

For the DST:
- I think you can remove either the `sr-only` label or the `aria-label`.

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** [Search input component](https://design.va.gov/components/search-input)
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by allanto-ah** [comment]
Environment: Staging

Verified that the extra labeling has been removed
